### PR TITLE
Dr-1111 - Add plans page

### PR DESCRIPTION
**Pull request by mvillaseco**
#  Overview
This Pr will add the plans page for a user to be able to upgrade his account.

#  Changes:
 - Add Plan page, controller  and service.
 - Fix responsive issues.

# _Technical debt:_
 - Test.

### CODE REVIEW

File: src/wwwroot/partials/header.html [hunk 1]
```diff
@@ -30,7 +30,8 @@ <h4 ng-if="isPermanentAuthed()"> {{'downloads_header' | translate}} </h4>
       </a>
     </nav>
     <nav ng-if="isPermanentAuthed()">
-      <div click-outside="hideConfigDropDown()">
+      <div click-outside="hideConfigDropDown()" class="right-container">
+        <a href="#/settings/my-plan" class="upgrade--link uppercase"><strong>Upgrade</strong></a>
```
Comment (andresmoschini) [inline src/wwwroot/partials/header.html]: It should be hidden when the user is not free, also the URL should throw an error if the user is not free.

File: src/wwwroot/controllers/PlanCtrl.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    }
+
+    function loadPlans() {
+      return settings.getPlansAvailable()
+      .then(function(response) {
+        planItems = response.data;
+      });
+    }
+
+    updatePlan(defaultPlanId);
```
Comment (nreal) [inline src/wwwroot/controllers/PlanCtrl.js]: This line es correct?  Because I do not understand

File: src/wwwroot/partials/header.html [hunk 3]
```diff
@@ -30,7 +30,8 @@ <h4 ng-if="isPermanentAuthed()"> {{'downloads_header' | translate}} </h4>
       </a>
     </nav>
     <nav ng-if="isPermanentAuthed()">
-      <div click-outside="hideConfigDropDown()">
+      <div click-outside="hideConfigDropDown()" class="right-container">
+        <a href="#/settings/my-plan" class="upgrade--link uppercase"><strong>{{'upgrade_text' | translate}}</strong></a>
```
Comment (nreal) [inline src/wwwroot/partials/header.html]: Is possible remove `<strong>` tag with an existence class?

File: src/wwwroot/services/settings.js [hunk 4]
```diff
@@ -121,5 +122,16 @@
         return response.data.api_keys;
       });
     }
+
+    function getPlansAvailable() {
+      var url = RELAY_CONFIG.baseUrl
+        + '/plans/';
```
Comment (nreal) [inline src/wwwroot/services/settings.js]: I think that last `/` is not necessary

**Comment by nreal** [comment]
👍

**Comment by andresmoschini** [comment]
Nice work @mvillaseco ! remember to remove all the links to the new page before merge.

**Pull request merged**